public/plugin.tsx

```diff
@@ -157,8 +204,21 @@ export class SecurityPlugin implements Plugin<SecurityPluginSetup, SecurityPlugi
     if (config.multitenancy.enabled) {
       addTenantToShareURL(core);
     }
+
+    if (config.multitenancy.enable_aggregation_view) {
+      const columns = deps.savedObjectsManagement.columns.getAll();
+    }
+
     return {};
   }
 
   public stop() {}
 }
+
+function isPrivateTenant(selectedTenant: string) {
+  return selectedTenant.startsWith('__user__');
+}
+
+function isGlobalTenant(selectedTenant: string) {
```

Comment:
we can default this to null incase nothing is passed by `selectedTenant: string = null`

Reply:
Great point! `selectedTenant: string = null` would not work, but I've changed it to handle the `null` case!